### PR TITLE
[KYIV-2022] add Kyiv 2022

### PR DESCRIPTION
**Pull request by delgod**
🎉 we are going hybrid!

### CODE REVIEW

File: content/events/2022-kyiv/welcome.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
++++
+Title = "devopsdays Kyiv 2022"
+Type = "welcome"
+aliases = ["/events/2022-kiev", "/events/2022-kyiv/"]
+Description = "devopsdays Kyiv 2022"
++++
+
+### Yes, we are going offline!
```
Comment (phrawzty) [inline content/events/2022-kyiv/welcome.md:8]: Offline or hybrid (re: your PR comment)?

File: data/events/2022-kyiv.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+# Location
+#
+coordinates: "50.449379,30.5209386" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Kyiv" # Defaults to city, but you can make it the venue name.
+#
+location_address: "52, Antonovycha str, Kyiv, Ukraine" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
```
Comment (phrawzty) [inline data/events/2022-kyiv.yml:39]: You may wish to keep this commented out until `$cfp_date_start`, or you'll risk getting submissions before you plan on it.

File: data/events/2022-kyiv.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    url: https://forms.gle/h9wxjHaRk1EQ6Kyw9
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
+  - name: contact
+  - name: conduct
+  - name: devopsdays.com.ua
+    url: https://devopsdays.com.ua/
```
Comment (phrawzty) [inline data/events/2022-kyiv.yml:49]: Unrelated to this PR, but this site still has your 2021 event…

File: content/events/2022-kyiv/welcome.md (hunk 1 above)
Comment (delgod) [reply content/events/2022-kyiv/welcome.md:8]: We want to promote as offline. I will change wording if needed

File: data/events/2022-kyiv.yml (hunk 2 above)
Comment (delgod) [reply data/events/2022-kyiv.yml:39]: thanks! I have fixed `cfp_date_start`

File: data/events/2022-kyiv.yml (hunk 3 above)
Comment (delgod) [reply data/events/2022-kyiv.yml:49]: thanks, we know, it is in progress, should be fixed a bit later

**Pull request merged**